test/linalg/lapack.jl

```diff
@@ -141,8 +141,8 @@ end
 
 @testset "gelsy, gelsd" begin
     @testset for elty in (Float32, Float64, ComplexF32, ComplexF64)
-        A = rand(elty,10,10)
-        B = rand(elty,10,10)
+        A = 10I + rand(elty, 10, 10)
+        B = 10I + rand(elty, 10, 10)
         C, j = LAPACK.gelsd!(copy(A),copy(B))
         D, k = LAPACK.gelsy!(copy(A),copy(B))
         @test C ≈ D
```

Comment:
I'd prefer to have `@test C ≈ D rtol=cond(A)` and continue to have some variation in the condition number of `A`.

Reply:
Updated: forgot the `eps`

Reply:
Cheers, and done! :)